content/guides/core-concepts/retry-ability.md

```diff
@@ -263,9 +263,19 @@ cy.get('#ssr-error', { timeout: 0 }).should('not.exist')
 
 Here is a short test that demonstrates some flake.
 
-```javascript
+:::visit-mount-test-example
+
+```js
+cy.visit('/')
+```
+
+```js
+cy.mount(<MyComponent />)
```

Comment:
Please rename this component to be more relevant

Reply:
missed that one 👍